src/message/messageActionSheet.js

```diff
@@ -52,15 +58,15 @@ const reply = ({ message, dispatch, ownEmail }) => {
 reply.title = 'Reply';
 reply.errorMessage = 'Failed to reply';
 
-const copyToClipboard = async ({ _, auth, message }) => {
+const copyMessageContent = async ({ _, auth, message }) => {
   const rawMessage = isAnOutboxMessage(message) /* $FlowFixMe: then really type Outbox */
     ? message.markdownContent
     : (await api.getRawMessageContent(auth, message.id)).raw_content;
```

Comment:
In the last commit when `getRawMessageContent` gets introduced, this spot should switch to it. Basically that new function is the result of factoring out this statement.